catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorKeyboardTest.java

```diff
@@ -62,53 +59,92 @@
 
 	@Before
 	public void setUp() throws Exception {
-		createProject("formulaEditorKeyboardTest");
+		Script script = BrickTestUtils.createProjectAndGetStartScript("FormulaEditorListFragmentTest");
+		script.addBrick(new ChangeSizeByNBrick(0));
 		baseActivityTestRule.launchActivity(null);
 	}
 
 	@Category({Cat.AppUi.class, Level.Smoke.class})
 	@Test
 	public void numericKeysTest() {
-		onView(withId(R.id.brick_set_variable_edit_text)).perform(click());
-
-		onView(withId(R.id.formula_editor_keyboard_1)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_2)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_3)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_4)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_5)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_6)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_7)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_8)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_9)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_0)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_decimal_mark)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_1)).perform(click());
-		onView(withId(R.id.formula_editor_keyboard_ok)).perform(click());
+		onBrickAtPosition(0)
+				.checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(1)
+				.checkShowsText(R.string.brick_change_size_by);
+		onBrickAtPosition(1)
+				.onChildView(withId(R.id.brick_change_size_by_edit_text))
+				.perform(click());
+
+		onView(withId(R.id.formula_editor_keyboard_1))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_2))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_3))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_4))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_5))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_6))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_7))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_8))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_9))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_0))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_decimal_mark))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_1))
+				.perform(click());
+		onView(withId(R.id.formula_editor_keyboard_ok))
+				.perform(click());
 
 		onView(withId(R.id.formula_editor_edit_field))
```

Comment:
theres now a conflict with this file, when resolving this please do use the changes from dev! also i know that im aware that i established the code styles for espresso and keep digging on them, but in this special case, since its always "click", and always on the same kind of element, it actually decreases readability when you add linebreaks.

with the resulting check below, weve got a 80char limit per line, and im sure this ones is now too long.

when you do have new tests PRs like this, please do not just go there and touch everything and perform style changes all over the place. if you want to do that open a separate PR for those